### PR TITLE
Yean

### PR DESCRIPTION
**Pull request by augisticz**
1. Objective
==> change text

2. Description of change
==> change text "Hello world" to "Hello worlds"

3. Quality assurance
==> N/A

4. Operation Impact
==> N/A

5. Business Impact
==> N/A

6. Priority of change
==> N/A

### CODE REVIEW

File: yean.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <script>
-	console.log('hello world')
+	console.log('hello worlds')
+
```
Comment (iporsut) [inline yean.js]: Space เว้นไว้ทำไม

Comment (iporsut) [inline yean.js]: Space เว้นไว้ทำไม

**Review by iporsut — CHANGES_REQUESTED** [review]

**Review by mixxuptangle — APPROVED** [review]
ผ่าน !!

**Review by iporsut — APPROVED** [review]

**Review by pipat2468 — APPROVED** [review]
..!

**Pull request merged**